### ISSUE TITLE
Peer dependency error with @okta/okta-auth-js v6

### ISSUE THREAD

**Issue by jonrimmer** [okta/okta-react#197]
## I'm submitting a:

- [x] Bug report  
- [ ] Feature request
- [ ] Other (Describe below)

## Current behavior

@okta/okta-auth-js v6 is now out on npm, but it is not specified as a valid peer dependency of `@okta/okta-react`. Trying to install the latest version of each will give the following error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: @okta/okta-auth-js@6.0.0
npm ERR! node_modules/@okta/okta-auth-js
npm ERR!   @okta/okta-auth-js@"^6.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @okta/okta-auth-js@"^5.3.1" from @okta/okta-react@6.4.1
npm ERR! node_modules/@okta/okta-react
npm ERR!   @okta/okta-react@"^6.0.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
```

## Expected behavior

The latest version of `@okta/okta-react` should work with the latest version of `@okta/okta-auth-js`.

## Minimal reproduction of the problem with instructions

Run:

```
npm i @okta/okta-auth-js @okta/okta-react
```

## Environment

- Package version: 6.4.1
- React version: N/A
- Browser: N/A
- OS: macOS
- Node version (`node -v`): v16.13.1
- Other: npm version 8.1.2

## Configuration

- Okta application type:

**Comment by oleksandrpravosudko-okta** [comment]
Thanks for the report @jonrimmer.

We are aware of peerDependencies issue and will release a fix soon.

Suggested workaround for now is to specify `--legacy-peer-deps` flag during package installation. Apologies for inconvenience.

**Comment by shuowu** [comment]
@jonrimmer Fix has been released in v6.4.2, closing this issue, feel free to reopen it if the issue still exist.
